pkg/util/log/log.go

```diff
@@ -383,6 +384,13 @@ func Warnf(format string, params ...interface{}) error {
 func Errorf(format string, params ...interface{}) error {
 	if logger != nil && logger.inner != nil && logger.shouldLog(seelog.ErrorLvl) {
 		return logger.errorf(format, params...)
+	} else if logger == nil || logger.inner == nil {
+		// We're currently trying to log an error before initializing
+		// the log module. This meant a early error while starting the
+		// agent. We should not silence such error.
+		err := fmt.Errorf(format, params...)
```

Comment:
Should we still scrub this to avoid potential credential leakage (same below)?

Reply:
Good question. We print stderr from the executable if it exits with a status code different from 0.

I would say it's up to the user to avoid having any secret in STDERR. What do you think ?

The doc already says (https://github.com/DataDog/datadog-agent/blob/master/docs/agent/secrets.md#retrieving-secrets-from-the-secret-backend):
```
Never output sensitive information on STDERR. If the binary exit with a different status code than 0 the agent will log the standard error output of the executable to ease troubleshooting.
```

Reply:
I think we should keep the scrubbing. Mainly because there is potential for other scenarios where your codepath could be hit where the developer might be expecting `log` to perform auto-scrubbing....

Reply:
:+1: